spacetx/starfish
# Travis cleanup (see 596)

* add test (`make check_notebooks`) for untested notebooks
 * add help to makefile
 * add check_requirements
 * re-simplify .travis.yml
 * implement other suggestions of gh-596

Testing plan:
 * check `make help` is pleasing to the eye
 * add a `new notebooks/py/*.py` and check that `make check_notebooks` fails
 * review the fact that PRs *don't* run notebooks (a subset?)
 * comment on the checks that should happen in cron (like `check_requirements`)
   - include docs?

closes #605
